WireUI/Sources/WireMainNavigationUI/Models/MainSidebarMenuItem.swift

```diff
@@ -16,9 +16,11 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-public enum MainSidebarMenuItem: Sendable {
+import Foundation
+
+public enum MainSidebarMenuItem: Sendable, Equatable {
     // conversation filters
-    case all, favorites, groups, oneOnOne, archive
+    case all, favorites, groups, oneOnOne, archive, folder(id: UUID, name: String)
```

Comment:
Same here, since the sidebar doesn't show the folder's name, can't we get rid of the info?
```suggestion
    case all, favorites, groups, oneOnOne, archive, folders
```

Reply:
You're right. I'll fix it. Thanks!